Monika After Story/game/script-topics.rpy

```diff
@@ -4727,6 +4727,193 @@ label monika_write:
     m "Thanks for listening~"
     return
     
+init 5 python:
+    for key in ['feature','boobs','sight','personality']:
+        monika_topics[key].append('monika_standsout')
+    monika_random_topics.append('monika_standsout')
+
+label monika_standsout:
+    m 2a "Hey, [player]? What about me stands out the most to you?"
+    m "I know it's a random question. Maybe even a bit silly."
+    m 5a "But I'm dying to know."
+    menu:
+        m "So...?"
+        "Your beautiful, green eyes of course.":
+            m 4k "Aww, really?"
+            m 4e "Thank you, sweetheart."
+            m "I wish I could see your eyes."
+            m 1i "Actually, what color are your eyes?"
+            menu:
+                m 5a "I need to know now."
```

Comment:
So just learned recently that you cannot have dialogue with expressions right after a `menu` block. Basically, this works:
```
menu:
    m "works"
    ...
```
but this doesnt:
```
menu:
    m 5a "no worko"
    ...
```
If you want to keep the expression, move the line of dialogue before the `menu`. If you want to keep the text on the screen, remove the expression part of the line.

Reply:
Will fix.